src/Admin/Vue/PaymentMethods.php

```diff
@@ -272,7 +272,7 @@ public static function integrated_payment($method) {
 				"name"    => "payplug_integrated",
 				"label"   => __( 'payplug_section_standard_payment_option_integrated_label', 'payplug' ),
 				"value"   => "integrated",
-				"checked" => $method['integrated']
```

Comment:
because of the changes you did up, you'd needed to change this, but this is wrong. this will mean if you're saved the redirect as payment_method you'll see IP as checked...